niaiddata.org/manifest.json

```diff
@@ -73,11 +73,11 @@
     {
       "target-port": 8888,
       "cpu-limit": "1.0",
-      "memory-limit": "1024Mi",
-      "name": "Jupyter Notebook Bio Python",
-      "image": "quay.io/occ_data/jupyternotebook:fix_niaid-test",
-      "env": {"FRAME_ANCESTORS": "https://aids.niaiddata.org https://tb.niaiddata.org https://microbiome.niaiddata.org https://flu.niaiddata.org"},
-      "args": ["--NotebookApp.base_url=/lw-workspace/proxy/","--NotebookApp.password=''","--NotebookApp.token=''"],
+      "memory-limit": "14512Mi",
+      "name": "Jupyter Notebook TB Ariba and Mykrobe- Lab Edition",
```

Comment:
Please change to "Jupyter Notebook Predict Antimicrobial Drug Resistance of Tuberculosis"